.github/workflows/build-and-upload-archives-upon-creating-tags.yml

```diff
@@ -1,33 +1,24 @@
-name: Build and Upload Archives upon Creating Tags
+name: Build and Upload Archives Upon Creating Tags
 
-# Run this workflow every time a tag is created/pushed
+# Run this workflow every time a tag is created
 on:
-  push:
-    tags:
-      - '*'
+  create:
```

Comment:
nit: I don't think we need the `:` after create. The example on the docs doesn't have them